vme/src/structs.cpp

```diff
@@ -311,31 +311,275 @@ zone_type::~zone_type()
         FREE(spmatrix);
 }
 
-file_index_type::file_index_type()
+unit_data *file_index_type::find_symbolic_instance_ref(unit_data *ref, ubit16 bitvector)
 {
-    name = nullptr;
-    zone = nullptr;
-    // next = NULL;
-    // unit = NULL;
+    unit_data *u = nullptr;
+    unit_data *uu = nullptr;
 
-    filepos = 0;
-    length = 0;
-    crc = 0;
+    if (ref == nullptr)
+    {
+        return nullptr;
+    }
 
-    no_in_zone = 0;
-    no_in_mem = 0;
-    room_no = 0;
-    type = 0;
+    if (IS_SET(bitvector, FIND_UNIT_EQUIP))
+    {
+        for (u = UNIT_CONTAINS(ref); u; u = u->next)
+        {
+            if ((UNIT_FILE_INDEX(u) == this) && UNIT_IS_EQUIPPED(u))
+            {
+                return u;
+            }
+        }
+    }
 
-#ifdef DMSERVER
```

Comment:
Have you understood what the difference is here? I suppose this was compiled for the vme but not so for the vmc.

Reply:
So this is in the old ctor and only init's the value if its DMSERVER. It changed to always be init'ed to zero